_includes/header.html

```diff
@@ -20,7 +20,18 @@
             <h3 class="brand-heading">DevsKrate</h3>
             <!-- <h5>&nbsp;</h5> -->
             <span class="intro-text">{{ content }}</span>
-            
+            <!-- Social Buttons Start -->
+
+            <ul class="list-inline social-buttons">
```

Comment:
no need of this.
just {% include social-buttons.html %} 
this will take care of this.

https://github.com/devskrate/biowith/blob/master/_includes/footer.html#L5